pkg/log/context.go

```diff
@@ -36,3 +58,40 @@ func goid() string {
 	idField := strings.Fields(strings.TrimPrefix(string(buf[:n]), "goroutine "))[0]
 	return idField
 }
+
+func paramKey(name string) string {
+	return PARAM + name
```

Comment:
Why do we need to add a constant?

Reply:
since I am adding all route parameters in the context (for generality purposes) I prefixed them to not create collision with other log fields across the application